pom.xml

```diff
@@ -94,8 +94,10 @@
 		<package.group>${project.artifactId}</package.group>
 		<package.install.dir>/usr/share/${project.artifactId}</package.install.dir>
 		<package.user>${project.artifactId}</package.user>
+                <package.userId>102</package.userId>
```

Comment:
We could prevent specifing number, and letting system decide (with -r option), couldn't we?